### ISSUE TITLE
Presence of attachments not highlighted, when email ends with a lot of empty lines

### ISSUE THREAD

**Issue by dilyanpalauzov** [thunderbird/thunderbird-android#6615]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.400

### Where did you get the app from?

F-Droid

### Android version

irrelevant

### Device model

irrelevant

### Steps to reproduce

I receive an email with the structure
```
Content-Type: multipart/related; type="multipart/alternative"
  Content-Type: multipart/alternative
    Content-Type: text/plain
    Content-Type: text/html
    Content-Type: text/calendar
  Content-Type: image/png
```
K9 shows the text/html part.  It contains a lot of new lines at the end.  After the new lines, K9 shows that the email has two attachments: a png and a iCalendar file.  The text/html part does not mention the timeslot of the meeting, that is i the text/calendar invitation.

The problem report is, that K9 shows in the visible area the text/html part and the white space contained in that MIME-part.  Off screen (outside the view port) there is information that the email contains attachments.  However I as a user, do not perceive that the displayed email is shown incomplete, so I do not come to the idea to scroll down to see see that there are attachments.

### Expected behavior

Even if the email ends with a lot of white space, and the view port (displayed part of the email) contains the complete email, excluding some terminating new lines, I as user shall be aware that the email contains an attachment.

Under these circumstances currenlty it can be easily overseen that the email contains attachment.

### Actual behavior

I have replied to that email, that the meeting invitation (based only on its text/html MIME part) contains no timeslot.

**Comment by fereshtehamiri** [comment]
hello dilyanpalauzov
We have thoroughly tested and replicated the bug scenario, and it has occurred for us on the following device and software configuration:
Phone Model: Samsung A32
Android Version: 13
K9 App Version: 6.702

**Comment by cketti** [comment]
While not exactly the same, this case should be covered by whatever we'll to do to fix #2067.

Closing this as duplicate.
